ReactAndroid/src/main/java/com/facebook/react/views/image/MultiPostprocessor.java

```diff
@@ -52,9 +52,20 @@ public String getName() {
   public CacheKey getPostprocessorCacheKey() {
```

Comment:
have you check what are the side effects of this method returning null?

Reply:
Looking at the uses inside Fresco, `null` is an accepted and expected value:

https://github.com/facebook/fresco/search?p=2&q=getPostprocessorCacheKey&unscoped_q=getPostprocessorCacheKey